### DISCUSSION TITLE
Perimetric Alarm not making sound

### DISCUSSION THREAD

**Discussion by fortizen** [cedricp/ddt4all#1019] [Q&A]
Hello, 1 Year ago i enabled the Perimetric Alarm along with Panic Button on My 2019-2020 Renault Megane RS Trophy Ph1 but the car doesen't make sound, only the Indicators turn on. How do i fix it or change the Alarm Output speaker to the Beep horn which works when i move away from the Car and it locks?

It uses the Same High Pitched Horn because i saw Factory Enabled Alarms in Base Model Meganes use it.

**Comment by AndyIreland** [comment]
Hi, i dont know what version BCM/UCH you have on your particular car but have you checked this: 
![image](https://github.com/cedricp/ddt4all/assets/49172217/05b2f284-9abc-48f8-a759-323578c5715f)

then check these settings:
![image](https://github.com/cedricp/ddt4all/assets/49172217/2fadcf98-503c-44fa-8a6d-2bb251fd78ad)

even so, without the aftermarket renault alarm system (with its own horn) I thought when you make those setting the pedestrian horn (where you press the middle of the steering wheel) just sounds/blasts instead?
